tests/test-install-command.php

```diff
@@ -41,6 +41,44 @@ public function test_install_wordpress() {
 		$this->assertFileExists( "{$output}/new-site/wp-content/mu-plugins/new-site-loader.php" );
 	}
 
+	public function test_install_wordpress_dev() {
+		$output = __DIR__ . '/output';
+
+		if ( is_dir( $output . '/new-site-dev' ) ) {
+			exec( 'rm -rf ' . $output . '/new-site-dev' );
+		}
```

Comment:
Should this be part of a fixture for this test class?

Reply:
Good call :)